### DISCUSSION TITLE
FreeStyle Libre 2 sensor error (373) on synamic drops / rises

### DISCUSSION THREAD

**Discussion by Inkalika** [j-kaltes/Juggluco#118] [General]
FreeStyle Libre 2. Juggluco 7.4.0 is giving sensor error on rapid BG drops or when sugar level is growing fast. These are most critical monents. Is it because of sensor not sending data? Is is possible to improve this somehow, or there is simply no data from the sensor in such moments?
The first sensor transmitted readings with error 373 appearing regularly. After 1 hour of error occurrence, a second sensor was installed. After transmitting a few readings, the second sensor also displayed a sensor error.

Out of 10 sensors, this error appeared on all of them. on two sensors very often and sometimes the data did not arrive for about an hour.

And are there any time limits after which the readings will not be restored and the sensor must be replaced?

PS After another half an hour both sensors were scanned by NFC, but the Sensor Error is still displayed
![photo_2024-01-30_13-25-00](https://github.com/j-kaltes/Juggluco/assets/89869109/cd11f1f2-1fa7-4f16-bcfc-2bd0e7ac8e58)

**Comment by j-kaltes** [comment]
If you want to see a glucose value independent on how unreliable it is (or just a value of the previous day), you can scan a European Freestyle Libre 2 sensor with Glimp: https://play.google.com/store/apps/details?id=it.ct.glicemia

**Reply by jeduden** [reply]
Is there a way to see - the raw values?
In a lot of cases, rapid drops happen during intensive sport activities, when the IOB is a few units (too high).
In this scenario, showing "noisy" data is better than no data.
I have also experienced sensors when there was an actual sensor issue.
In both cases, I would like to see the raw data, and for example a confidence interval.
This enables me to make an informed decision not just based on sensor data but also using iob and activity level.
